airflow/providers/google/cloud/operators/vertex_ai/generative_model.py

```diff
@@ -0,0 +1,181 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This module contains Google Vertex AI Generative AI operators."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Sequence
+
+from airflow.providers.google.cloud.hooks.vertex_ai.generative_model import GenerativeModelHook
+from airflow.providers.google.cloud.operators.cloud_base import GoogleCloudBaseOperator
+
+if TYPE_CHECKING:
+    from vertexai.preview.generative_models import ChatSession
+
+    from airflow.utils.context import Context
+
+
+class PromptLanguageModelOperator(GoogleCloudBaseOperator):
```

Comment:
Can you please describe what is the purpose of having this as a separate operator in separate module? I think we have almost the same functionality in model_service.py so seems like we can extend it?